File: abc/abc126/abc126_a.cs
// https://atcoder.jp/contests/abc126/submissions/8399455

using System;
using System.Linq;

namespace CSharp
{
    class Program
    {
        public static void Main()
        {
            var input = Console.ReadLine().Split(' ').Select(int.Parse).ToArray();
            var K = input[1];

            var S = Console.ReadLine().ToCharArray();
            S[K - 1] = char.ToLower(S[K - 1]);
            Console.WriteLine(S);
        }
    }
}

File: abc/abc126/abc126_a.fs
// https://atcoder.jp/contests/abc126/submissions/12209156

let [| _; K |] = stdin.ReadLine().Split() |> Array.map int

stdin.ReadLine().ToCharArray()
|> Array.mapi (fun i x ->
    if i = K - 1 then ((string) x).ToLower().ToCharArray().[0]
    else x)
|> Array.iter stdout.Write
